src/components/inputs/Dates/DateRange/DateRange.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactElement } from 'react';
+import '@testing-library/jest-dom';
+import {
+  render,
+  RenderOptions,
+  screen,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { ThemeProvider } from 'styled-components';
+import { themes } from '../../../../themes';
+import { DateRange } from './DateRange';
+
+const Provider = ({ children }) => {
+  return (
+    <ThemeProvider theme={themes['light']}>{children}</ThemeProvider>
+  );
+};
+
+const renderWithThemeProvider = (
+  ui: ReactElement,
+  options?: Omit<RenderOptions, 'wrapper'>
+) => render(ui, { wrapper: Provider, ...options });
+
+describe('DateRange', () => {
+  it('Renders DateRange', () => {
+    const { container } = renderWithThemeProvider(<DateRange />);
+
+    expect(container).toBeInTheDocument();
+  });
+
+  fit('Selects range', async () => {
+    const minDate = new Date(2022, 8, 26);
+
+    // const setStateMock = jest.fn();
+    // const useStateMock = (useState) => [useState, setStateMock];
+    // jest.spyOn(React, 'useState').mockImplementation(useStateMock);
+
+    const { debug } = renderWithThemeProvider(
+      <DateRange
+        data-testid="calendar"
+        minDate={minDate}
+        // onChange={setStateMock}
+      />
+    );
+
+    const startDateInput = screen.getByRole('textbox', {
+      name: 'Start Date',
+    });
+    await userEvent.type(startDateInput, '09/27/2022');
+
+    const endDateInput = screen.getByRole('textbox', {
+      name: 'End Date',
+    });
+    await userEvent.type(endDateInput, '09/28/2022');
+
+    debug(endDateInput);
```

Comment:
Both dates appear in this input value for some reason: '09/27/202209/28/2022'

Reply:
Looks like they're separate elements... 🤔 : 
![Screen Shot 2022-09-28 at 12 37 38 PM](https://user-images.githubusercontent.com/22207955/192838364-138d1506-7ea3-496a-9162-d3f568d797cb.png)